services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java

```diff
@@ -142,6 +142,16 @@ public static Builder builder() {
   @Nullable
   public abstract String requestUuid();
 
+  /*
+   * List of calculated toll costs for the route. See `TollCost`.
+   *
+   * return list of toll costs
+   */
```

Comment:
@dzinad hey! Why have you used  `multi-line` comment instead of `Javadoc` comment? Previously we always used `Javadoc`, so new comments don't looks consistent with old on a customer side.

<img width="748" alt="image" src="https://user-images.githubusercontent.com/14282440/189054857-e3021cb2-2df0-4a0f-855a-95bc028139a1.png">

Reply:
ah, looks like we don't need them to be documented

Reply:
yes, this is expected